vendor/github.com/heimweh/go-pagerduty/pagerduty/pagerduty.go

```diff
@@ -52,6 +52,7 @@ type Client struct {
 	ServiceDependencies *ServiceDependencyService
 	Priorities          *PriorityService
 	ResponsePlays       *ResponsePlayService
+	Tags                *TagService
```

Comment:
Gut check -- is adding tags intentional? Not mentioned in the PR description.

Reply:
removed Tags from client for now.